WordPress/Classes/Extensions/UIImageView+SiteIcon.swift

```diff
@@ -108,6 +109,14 @@ extension UIImageView {
     @objc
     func downloadSiteIcon(for blog: Blog, placeholderImage: UIImage? = .siteIconPlaceholder) {
         guard let siteIconPath = blog.icon, let siteIconURL = optimizedURL(for: siteIconPath) else {
+
+            let standardSize = CGSize(width: SiteIconDefaults.imageSize, height: SiteIconDefaults.imageSize)
```

Comment:
Nitpick: since `standardSize` is only used inside the following `if`, maybe move it in there? Or just nix it altogether and put the `CGSize` in the `UIImage` creation.

`image = UIImage.gridicon(.p2, size: CGSize(width: SiteIconDefaults.imageSize, height: SiteIconDefaults.imageSize))`